jlm/mlir/frontend/MlirToJlmConverter.cpp

```diff
@@ -237,106 +270,160 @@ MlirToJlmConverter::ConvertFPBinaryNode(
       { inputs[0], inputs[1] });
 }
 
+llvm::fpcmp
+MlirToJlmConverter::ConvertFPCMP(const ::mlir::arith::CmpFPredicate & op)
```

Comment:
It would be nice to combine this with the converter that goes the opposite direction, and use a `util::BijectiveMap` that is used in both directions. The common map and converter functions could be placed in a `MlirConverterCommon.[ch]pp` placed outside both the fronted and backend folder

Reply:
In the new cpp file you make